docs/platform-support.rst

```diff
@@ -215,3 +215,33 @@ NVIDIA Network Operator has been validated in the following scenarios:
      - Yes
      - 
 
+
+=============================================
+Supported Precompiled Images for DOCA Drivers
+=============================================
+
+--------
+Overview
+--------
+
+To save startup (compilation) time and operational effort, precompiled DOCA driver images are available for common OS/flavor/kernel variants.
+
+The container image tag pattern used for common variants is: **driver_ver-container_ver-kernel_ver-flavor-os**. For example: ``24.07-0.6.1.0-6.8.0-49-generic-ubuntu24.04``
+
+**NOTE:** The tags are multi-architecture manifests, so no need to state the architecture.
```

Comment:
are you going to propose a corresponding change to network operator? precompiled image tag requires CPU architecture now [1]

[1] https://github.com/Mellanox/network-operator/blob/e44a4bd19135374afb5b0aab32e2af2c688751cc/pkg/state/state_ofed.go#L68

Reply:
Oh, I wasn't aware of that...

The simplest workaround would be for me to skip creating manifests, and just deliver both architecture tags explicitly (but that'd be a step backwards)...
Alternatively, how simple/quick would it be to adjust the `state_ofed.go` code to make the `-<cpu-arch>` suffix _optional_? (i.e. if it's not stated then to assume an appropriate multi-arch manifest is provided)

Reply:
Ok, the tags with explicit architecture are still pushed so this code/automation should work fine.

And for cases where a user may want to explicitly/manually set the DOCA version - then they may choose to use the manifest tag (which points to both architectures).